index.html

```diff
@@ -36,7 +36,8 @@
 
                 <div class="masthead clearfix">
                     <div class="inner">
-                        <h3 class="masthead-brand"><a href="https://rikumax25.github.io/">3SDSetup</a></h3>
+                        <h3 class="masthead-brand"><a href="https://rikumax25.github.io/3SDSetup/">3SDSetup</a></h3>
+						<h3 class="masthead-git"><a href="https://github.com/rikumax25/3SDSetup/">Source</a></h3>
```

Comment:
i dont know why the spacing is showing up like this online, in notepad++ it looks fine